var/spack/repos/builtin/packages/py-psyclone/package.py

```diff
@@ -34,12 +36,10 @@ class PyPsyclone(PythonPackage):
     version("2.0.0", sha256="94766ffda760404af99f85d70341376192e4a1b8e16e7ae5df980038898a9c41")
     version("1.5.1", sha256="f053ad7316623b2a4002afc79607abda3b22306645e86f2312d9f3fe56d312dc")
 
-    depends_on("fortran", type="build")  # generated
-
     # Current dependencies
     depends_on("py-setuptools", type="build")
     depends_on("py-pyparsing", type=("build", "run"))
-    depends_on("py-fparser@0.1.4:", type=("build", "run"), when="@2.5.0")
+    depends_on("py-fparser@0.2.0:", type=("build", "run"), when="@3.0.0")
     depends_on("py-configparser", type=("build", "run"))
     depends_on("py-jsonschema", type=("build", "run"), when="@2.5.0")
```

Comment:
`py-jsonschema` and `py-jinja2` are also required for the new version per `setup.py`:

```
        install_requires=['pyparsing', 'fparser>=0.2.0', 'configparser',
                          'jsonschema', 'sympy', "Jinja2", 'termcolor',
                          'graphviz'],
```

Reply:
Oops again, sorry. Fixed now.